awkward/persist.py

```diff
@@ -556,7 +714,7 @@ def save(file, array, name=None, mode="a", **options):
     if isinstance(file, str) and not file.endswith(".awkd"):
         file = file + ".awkd"
 
-    alloptions = {"delimiter": "-", "suffix": ".raw", "schemasuffix": ".json", "compression": compression}
+    alloptions = {"delimiter": "-", "suffix": ".raw", "schemasuffix": ".json", "compression": True}
```

Comment:
Wait—how does `compression=True` make any sense? The compression option-handlers don't recognize `True` as an option. The default should be the default policy (`awkward.persist.compression`) and if `True` is accepted as a possible value, it should always compress (i.e. stricter than the default policy).

Reply:
It does not make sense. I've reverted it to the original behavior.